src/modal/testing/modal.service.spec.ts

```diff
@@ -7,17 +7,17 @@ import { BsModalService, ModalModule } from '../index';
 @Component({ template: '<div>Dummy Component</div>' })
 class DummyComponent {
   // eslint-disable-next-line @typescript-eslint/no-empty-function,@typescript-eslint/no-unused-vars
-    constructor(modalService: BsModalService) { }
+  constructor(modalService: BsModalService) { }
 }
 
 @Component({
-    template: '<div>Test Component</div>'
+  template: '<div>Test Component</div>'
 })
-class TestModalComponent { }
+export class TestModalComponent { }
```

Comment:
This should not be exported out of the test but it seems angular service call out an error here without the export. I can remove it if this is a concern.